screens/User/Profile.js

```diff
@@ -27,7 +27,11 @@ export default function Profile({ navigation }) {
           <Text fontWeight='semibold'>{currentUser.displayName}</Text>
           <Text>{currentUser.readerType}</Text>
           <Text fontStyle='italic' fontSize='sm'>
-            joined {new Date(currentUser.createdAt).toISOString().substring(0, 10).replaceAll('-', '/')}
+            joined{' '}
```

Comment:
This thing usually comes up with formatter. Be carefull next time

Reply:
What It mean is that to add a space in UI and I think you can add the space normally.